package-lock.json

```diff
@@ -22019,16 +22092,6 @@
 						"yauzl": "^2.7.0"
 					},
 					"dependencies": {
```

Comment:
```suggestion
					"dependencies": {
						"are-we-there-yet": {
 							"version": "2.0.0",
 							"resolved": "https://registry.npmjs.org/are-we-there-yet/-/are-we-there-yet-2.0.0.tgz",
 							"integrity": "sha512-Ci/qENmwHnsYo9xKIcUJN5LeDKdJ6R1Z1j9V/J5wyq8nh/mYPEpIKJbBZXtZjG04HiK7zV/p6Vs9952MrMeUIw==",
 							"dev": true,
 							"requires": {
 								"delegates": "^1.0.0",
 								"readable-stream": "^3.6.0"
 							}
 						},
```

Reply:
I hope this is all we need to make CI happy.

Reply:
GitHub doesn't let me apply your suggestion...
The browser console shows an error whenever I click the "Commit suggestion" button.

Reply:
I applied this patch locally: fdf6975